.devops/core-code-review-pipelines.yml

```diff
@@ -0,0 +1,73 @@
+pr:
+  autoCancel: false # prevent terraform state lock on new commits
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - "src/core/*"
+      - ".devops/*"
+
+parameters:
+  - name: "PROD"
+    displayName: "Run on PROD environment"
+    type: boolean
+    default: True
+    values:
+      - False
+      - True
+  - name: "CORE"
+    displayName: "Run on core domain"
+    type: boolean
+    default: True
+    values:
+      - False
+      - True
```

Comment:
since this is a CORE pipeline, we can remove the parameter

Reply:
yes sure